src/components/MainPage/index.js

```diff
@@ -0,0 +1,14 @@
+import { Component } from "react";
+
+class MainPage extends Component {
+  render () {
+
+    return (
+      <div>
+        <button> <a href="/counter">Go To Counter</a></button>
```

Comment:
Create a method in  which will implement redirecting to different routes. 
Never use `a` tag for navigation in your apps.
HINT: you can use `useHistory`  hook hear or use Link component from `react-router-dom` lib.
Link API - https://reactrouter.com/web/api/Link